fog/overseer/server/src/metrics/utils.rs

```diff
@@ -6,6 +6,11 @@ use mc_common::logger::{log, Logger};
 use mc_fog_api::ingest_common::{IngestControllerMode, IngestSummary};
 use std::{collections::HashSet, convert::TryInto};
 
+pub fn increment_unresponsive_node_count(logger: &Logger) {
+    log::trace!(logger, "Setting unresponsive node metric.");
+    counters::UNRESPONSIVE_NODE_COUNT.inc();
+}
+
```

Comment:
This is being logged in the worker.

Reply:
I'm sorry, I didn't mean log, I meant the prometheus metric capturing - the unresponsive node could potentially annotated with the prometheus peer_counter so that they show up on the metrics dashboard (separate from the logstash logs) with the node name.

https://github.com/mobilecoinfoundation/mobilecoin/blob/master/util/metrics/src/op_counters.rs#L89-L97

Reply:
Ah I see. I think this is something we can consider implementing in a follow-up PR if ops wants this info to be captured.